scripts/add-crowdin-contributors.ts

```diff
@@ -1064,21 +1064,27 @@ const list: any[] = [
   },
 ];
 
-const infoPath = path.join(__dirname, '..', '.all-contributorsrc');
-const info = fs.readJSONSync(infoPath);
-for (const user of list) {
-  if (user.login) {
-    info.contributors = allContributors.addContributorWithDetails({
```

Comment:
The call to `allContributors.addContributorWithDetails()` returns a pending promise and doesnt do anything. I changed it to be resolved with `Promise.all()`